runtime/compiler/env/J9SharedCache.hpp

```diff
@@ -418,6 +418,39 @@ class TR_J9SharedCache : public TR_SharedCache
     */
    static void buildWellKnownClassesSCCKey(char *buffer, size_t size, unsigned int includedClasses);
 
+    /**
+    * \brief Store the dependencies of an AOT method in the SCC
+    *
+    * The dependencies of an AOT method are encoded as an array of uintptr_t values. The first entry is the number
+    * of elements in the entire array. The subsequent entries are encoded offsets to the class chains of classes that
+    * need to be loaded or initialized before the compiled body can be loaded. The entry will be the offset itself
+    * if the class must be initialized, and it will be (offset & ~1) if the class only needs to be loaded.
```

Comment:
I don't understand the comment. Do you erase the last bit or set the last bit?

Reply:
I erase the last bit. Class chain offsets always have their last bit set. I've clarified.